release/models/network-instance/openconfig-evpn.yang

```diff
@@ -1061,5 +1068,165 @@ module openconfig-evpn {
 
       uses evpn-vxlan-parameters-config;
     }
+
+    container endpoint-peers {
+      description "Overlay endpoint peers oper data";
```

Comment:
It would be useful to have a better description here -- i.e., what is an overlay endpoint here?

Reply:
Done